### DISCUSSION TITLE
Help with the EDX seminar: Introduction to Cloud Development

### DISCUSSION THREAD

**Discussion by kopitsa** [community/community#31723] [New to GitHub]
Hello, to everyone. 
I am trying, to complete my on line seminar, but i am having problem with the Git Hub push. 
Specifically, i create my code to VS, and i tried multiple times, to give a push, to my Git Hub account. The result, is: my repository, not found. 
Please help me with that.  
I am spending my days, watching tutorials, trying to figure out a solution.
With regards 
Konstantina.

**Comment by jfullstackdev** [comment]
do you have that repo on GitHub ? you should create it first there to be sure

**Comment by kopitsa** [comment]
Thank you, for your respond. I have already the repo, on my profile, but i
cannot modified it.

Στις Παρ, 2 Σεπ 2022, 4:28 μ.μ. ο χρήστης jdevstatic <
***@***.***> έγραψε:

**Comment by jfullstackdev** [comment]
maybe some of the answers here might help 

https://stackoverflow.com/questions/10116373/git-push-error-repository-not-found

**Reply by kopitsa** [reply]
Thank you so much, for your time. 
I use some of the ideas, but still nothing. I notice that the Git Hub, is locked in a different repository. 
So i addressed, to a tech company, to help me out with it. 
with regards
Konstantina.
